server/starters/web-starter/src/main/java/com/oceanbase/odc/config/SecurityBeanConfiguration.java

```diff
@@ -73,4 +77,11 @@ public UsernamePasswordConfigureHelper usernamePasswordConfigureHelper() {
                 sensitivePropertyHandler, customAuthenticationSuccessHandler, customAuthenticationFailureHandler,
                 commonSecurityProperties);
     }
+
+    @Bean
+    public UsernamePasswordConfigureHelper jwtUsernamePasswordConfigureHelper() {
```

Comment:
there are two  UsernamePasswordConfigureHelper

Reply:
use conditional annotation, and add this annotation on customAuthenticationSuccessHandler and customJwtAuthenticationSuccessHandler